### PR TITLE
docs: Corrected first sample code so it runs

### PR DESCRIPTION
**Pull request by trgardos**
The first sample code won't run because it tries to create the browser window before app is ready.

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] PR release notes describe the change in a way relevant to app-developers

#### Release Notes

Notes:

### CODE REVIEW

File: docs/api/browser-window.md [hunk 1]
```diff
@@ -6,21 +6,26 @@ Process: [Main](../glossary.md#main-process)
...
-// Or load a local HTML file
-win.loadURL(`file://${__dirname}/app/index.html`)
+  // Load a remote URL
+  win.loadURL('https://github.com')
+
+  // Or load a local HTML file
+  win.loadURL(`file://${__dirname}/app/index.html`)
+
+  win.on('closed', () => {
+  win = null
```
Comment (jkleinsc) [inline docs/api/browser-window.md]: This line needs to be properly indented in order to pass our linting tests.  You can verify the change locally by making the change and running `npm run lint:docs`
```suggestion
    win = null
```

Comment (trgardos) [reply docs/api/browser-window.md]: Done. Thanks!

**Comment by codebytere** [comment]
Still seeing some linting issues:
```
2018-12-21T19:48:01.1744266Z    api/browser-window.md
2018-12-21T19:48:01.1745463Z          17:1:     Expected indentation of 2 spaces but found 4.
2018-12-21T19:48:01.1745761Z          19:1:     Expected indentation of 2 spaces but found 4.
2018-12-21T19:48:01.1746050Z          20:1:     Expected indentation of 2 spaces but found 4.
2018-12-21T19:48:01.1746270Z          22:1:     Expected indentation of 2 spaces but found 4.
2018-12-21T19:48:01.1746533Z          23:1:     Expected indentation of 2 spaces but found 4.
2018-12-21T19:48:01.1747158Z          25:1:     Expected indentation of 2 spaces but found 4.
2018-12-21T19:48:01.1768069Z          27:1:     Expected indentation of 2 spaces but found 4.
```

**Comment by jkleinsc** [comment]
@trgardos your original indentation of 2 spaces was correct.  The only indentation that was originally wrong was this:
```js
     win.on('closed', () => {
     win = null
     })
```
Should be:
```js
     win.on('closed', () => {
       win = null
     })
```

**Comment by BinaryMuse** [comment]
I think I'm 👎 on this change as well, especially given that we mention using the remote module in the example, where the `app.on('ready')` code wouldn't be relevant.

**Pull request closed without merging**